pkg/apis/api/v1alpha1/astarte_types.go

```diff
@@ -242,6 +242,33 @@ type AstarteRabbitMQSpec struct {
 	EventsExchangeName string `json:"eventsExchangeName,omitempty"`
 }
 
+type AstarteCassandraSSLConfigurationSpec struct {
+	Enabled bool `json:"enabled"`
+	// +optional
+	CustomCASecret v1.LocalObjectReference `json:"customCASecret,omitempty"`
+	// +optional
+	SNI *bool `json:"sni,omitempty"`
+	// +optional
+	CustomSNI string `json:"customSNI,omitempty"`
+}
+
+type LoginCredentialsSecret struct {
```

Comment:
This is by all means a duplicate of `AstarteRabbitMQConnectionSecretSpec`. But let's address this after the merge.